apsuite/commissioning_scripts/meas_coupling_tune.py

```diff
@@ -78,6 +82,10 @@ def ismeasuring(self):
         return self._thread.is_alive()
 
     def _do_meas(self):
+        if self.devices['quad'].devname != self.params.quadfam_name:
```

Comment:
This will always be `False`. The attribute `devname` is the complete name of the device. In order to get what you want here you need to compare the variable `quadfam_name` with the attribute `dev` of the `SiriusPVName` object.